codecombat/codecombat
# Small easy League page i18n improvements

# Context

The /league route is almost entirely hard coded english.
There are some tricky elements on the page to integrate with i18n.

The aim of this PR is to make a start at translating the esports page without tackling some of the more tricky blocks of text.

In a future PR we'll need to improve i18n for:
 - Headings that are broken into multiple colors, i.e. "COMPETITIVE CODING HAS NEVER BEEN SO EPIC"
 - Dynamic text pulled from a clan, i.e Team DerBezt
 - Headings that are images, i.e. "UNLIMITED POTENTIAL"


# Risk

Low risk. A possible typo could cause text to not show up properly.
I've reduced risk by manually testing.

# Testing process

1. Spin up local proxy with `npm run dev` and `npm run proxy`.
2. Navigate to localhost:3000/league
3. Expect all the text to appear in English correctly.
